substratevm/src/com.oracle.svm.core/src/com/oracle/svm/core/jdk/TimeZoneSubstitutions.java

```diff
@@ -78,12 +78,23 @@ public static TimeZone getTimeZone(String id) {
     static class Options {
         @Option(help = "When true, all time zones will be pre-initialized in the image.")//
         public static final HostedOptionKey<Boolean> IncludeAllTimeZones = new HostedOptionKey<>(false);
+
+        @Option(help = "A comma separated list of time zones that will be pre-initialized in the image.")//
+        public static final HostedOptionKey<String> IncludeTimeZones = new HostedOptionKey<>(null);
```

Comment:
I would also make this a `String[]`. That way you will be able to add time zones in different places on the command line (or jars) and they will compose (which I believe should be the right behavior).

Reply:
Done. I wasn't aware of this support. Thank you for pointing me to this. With this change, the param will now be passed (from command line) multiple times:

```
../../mx/mx native-image -H:IncludeTimeZones=Europe/Berlin -H:IncludeTimeZones=GMT HelloWorld 
```